src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -330,6 +428,55 @@ class TestRequestRateManager : public TestLoadManagerBase,
  private:
   bool use_mock_infer_;
 
+  void TestCustomDataSetup(std::vector<ModelTensor>& tensors)
+  {
+    std::shared_ptr<MockDataLoader> mdl{
+        std::make_shared<MockDataLoader>(params_.batch_size)};
+
+    std::shared_ptr<MockModelParser> mmp{
+        std::make_shared<MockModelParser>(on_sequence_model_, false)};
+    mmp->inputs_ = std::make_shared<ModelTensorMap>();
+    for (auto t : tensors) {
+      (*mmp->inputs_)[t.name_] = t;
```

Comment:
Why is this here? Doesn't this happen automatically somewhere else in PA?

Reply:
Under normal usage, the model parser `inputs_` would come from parsing the model's metadata that comes back from the triton server (which comes from the config.pbtxt).